File: fizzbuzz/6029562/fizzbuzz.py
x, y, n = map(int, input().split())

for i in range(1, n + 1):
	if i % x == 0:
		print('Fizz', end='')
	if i % y == 0:
		print('Buzz', end='')
	
	if not (i % x == 0 or i % y == 0):
		print(i, end='')
	print()

File: fizzbuzz/6375797/fizzbuzz.rb
require "readline"

x, y, n = gets.split().map(&:to_i)
for i in 1..n
  if i % x == 0 && i % y == 0
    puts "FizzBuzz"
  elsif i % x == 0
    puts "Fizz"
  elsif i % y == 0
    puts "Buzz"
  else
    puts i
  end
end